src/lib/rest/rest.cpp

```diff
@@ -1017,6 +1036,24 @@ static int connectionTreat
       ciP->httpHeaders.servicePath = defaultServicePath(url, method);
     }
 
+    char correlator[37];
```

Comment:
I'd suggest not using a 37 literal here. I'd suggest to put the CORRELATOR_ID_SIZE at limits.h or global.h (don't remberbe which one) and size arrays based on that.